nss-day-cohort-46/holiday-road-forever-genesis
# Added dependency chart

# Description

I added a screen shot of the dependency chart as it's own ```dependency.md``` file

Fixes # (issue)

## Type of change: Documentation


# Testing Instructions

```git fetch -- all
git checkout dm-settup
```

Check to see if the dependency.md file appears in the github repo



# Checklist:
